test/index.ts

```diff
@@ -55,3 +60,22 @@ function addEnvVarsToMochaOptions(options: Mocha.MochaOptions): void {
         }
     }
 }
+
+if (isWebpack) {
```

Comment:
Will help alert if we find ourselves in this situation again with picking up test files from the wrong location